leobot_web_server/web_content/scripts/vr.js

```diff
@@ -0,0 +1,6 @@
+var a = window.innerHeight;
+var b = window.innerWidth;
+var c = Math.round(b/2);
+console.log(a,b,c);
+document.getElementById('right_iframe').src = "http://0.0.0.0:8090/stream?topic=/leobot/stereocamera/right/image_raw&width="+c+"&height="+a;
```

Comment:
Hardcoded port. please either synchronize it with argument or remove argument and hardcode port number